executor/src/lib.rs

```diff
@@ -545,7 +545,8 @@ where
     pub fn execute_and_commit_chunk(
         &self,
         txn_list_with_proof: TransactionListWithProof,
-        ledger_info_with_sigs: LedgerInfoWithSignatures,
+        verified_target_li: LedgerInfoWithSignatures,
+        epoch_change_li: Option<LedgerInfoWithSignatures>,
```

Comment:
This isn't intuitive. No comment for what this field is and why it is an `Option`.